### ISSUE TITLE
Column organising fails on refresh

### ISSUE THREAD

**Issue by jarrydstan** [grokability/snipe-it#13640]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When rearranging the colums in any of  asset/software views, it works fine, but when i navigate to another page and then navigate back to that page, it goes back to the previous config and nothing is in the right spot.

Also when adding or removing columns after this, it doesnt update clearly.
![image](https://github.com/snipe/snipe-it/assets/65701829/63479fa7-c0aa-44e2-a36d-395d314e3fb5)

### Reproduction steps

1. Log in to dashboard
2. Navigate to any of the asset/license view
3. Re-arrange the columns
4. navigate to another page
5. navigate back to the previous page
6. everything is out of whack
...

### Expected behavior

When rearranging the columns, they should stay that way, either for the user or the entire instance.

When navigating back and forth, the state should stay the same.

### Screenshots

![image](https://github.com/snipe/snipe-it/assets/65701829/4fb6b757-3bd7-405b-a96e-fa64535ca393)

### Snipe-IT Version

latest v6.2.0-pre build 11531

### Operating System

Debian 11

### Web Server

Apache2

### PHP Version

7.4.33

### Operating System

Windows 10

### Browser

Chrome

### Version

116.0.5845.188

### Error messages

```shell
Error with Feature-Policy header: Unrecognized feature: 'document-domain'.
Error is reproduced on the demo.
No error in the stacktrace
No logs in storage/logs/ for some reason?
No errors in apache
[PHP Modules]
bcmath
calendar
Core
ctype
curl
date
dom
exif
FFI
fileinfo
filter
ftp
gd
gettext
gmp
hash
iconv
intl
json
ldap
libxml
mbstring
mcrypt
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
Phar
posix
readline
Reflection
session
shmop
SimpleXML
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlrpc
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache
```

### Additional context

Fresh install
OS is Debian 11, Web is Apache2
I manually installed 
No errors in installation, all worked ok
I had to change the root password, other than that nothing
It happens on the demo so i assume its just a bug

**Comment by snipe** [comment]
If you look at the original PR for this, I did notice it was a little “off”, but I did test refreshing and navigating away and it stuck. 

Can you try switching your BS tables to `localStorage`? If you have a lot of fields and sorting, etc, the cookie version can get overloaded and can cause weirdness

**Comment by jarrydstan** [comment]
Oh yeah! I just took a look at it, very strange, didn't realise it only just got added, this is my first install of snipe-it.

How do I add the tables to local storage sorry? I did a google search but nothing really comes up outside of writing JavaScript

**Comment by snipe** [comment]
@jarrydstan You'll want to set the `BS_TABLE_STORAGE` env var to be:

`BS_TABLE_STORAGE=localStorage`

https://snipe-it.readme.io/docs/configuration#optional-session-settings

<blockquote><img src="https://snipe-it.readme.io/favicon.ico" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/configuration#optional-session-settings">Configuration</a></strong></div><div>This section is where you edit the Snipe-IT configuration file to reflect your own settings, such as your database credentials, mail server, preferred language, timezone, and so on. Some of the settings are optional, some are required.Don't be intimidated by the length of this page. The configuratio...</div></blockquote>

<img width="798" alt="Screenshot 2023-09-25 at 11 33 57 AM" src="https://github.com/snipe/snipe-it/assets/197404/fa536d27-89b3-47e4-86bd-f5b680fd138d">

**Comment by jarrydstan** [comment]
Thanks snipe! I just changed that to localStorage and tested, the issue is still occurring :( Tried in edge as well as firefox

**Comment by snipe** [comment]
Yeah, I had to back that column re-order feature out on master :( It's the same damned conflict we've been having for years when we try to implement the extension. @uberbrady is trying to work through it to see if it's fixable.
